index.js

```diff
@@ -8,127 +8,192 @@ module.exports = {
         after: true,
         afterEach: true,
         Polymer: false,
-        moment: false
+        moment: false,
     },
     plugins: [
         'html',
-        'filenames'
+        'filenames',
     ],
     parserOptions: {
         ecmaVersion: 2018,
-        sourceType: 'module'
+        sourceType: 'module',
     },
     rules: {
-        'new-cap': [ 2, {
-            capIsNewExceptions: [ 'Polymer', 'Router' ]
+        // Possible Errors
+        'no-console': 'error',
+        'no-import-assign': 'error',
+        'no-inner-declarations': 'off',
```

Comment:
Where in the code does this break?

Reply:
This can be also "on", it doesn't break anything, as I can see.